model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -34,6 +33,45 @@ def create_layer(layer_dict):
     return sequential_dict
 
 
+def create_network(network_dict):
+    sequential_network = nn.Sequential()
+    for name, layer in network_dict.items():
+        if isinstance(layer, dict):
+            parameters = {k: v for k, v in layer.items()}
+            layer_type = parameters.pop("layer_type")
+            try:
+                if not isinstance(parameters["in_channels"], int):
+                    layers = []
+                    for module in sequential_network.children():
+                        layers.append(module)
+                    for module in reversed(layers):
+                        try:
+                            parameters["in_channels"] = module.out_channels
+                            break
+                        except:
+                            pass
+            except KeyError:
+                pass
+
+            try:
```

Comment:
I see that you have used multiple try except blocks. What are all the cases they are handling?

Reply:
Explained above

Reply:
We should refrain from using the try except blocks, and look for ways to handle all the cases without them. (As it makes the code less readable and might bypass errors that shouldn't go unnoticed in the future)